.circleci/config.yml

```diff
@@ -324,17 +365,20 @@ workflows:
   version: 2
   build-and-test:
     jobs:
-      - go1_12-build:
-          matrix:
-            parameters:
-              build_tags: [ "", "appsec" ]
       - metadata
       - lint
       - test-core:
           matrix:
             parameters:
+              gover: ["1.13", "1.14", "1.15", "1.16", "1.17"]
               build_tags: [ "", "appsec" ]
       - test-contrib:
           matrix:
             parameters:
+              gover: ["1.14", "1.15", "1.16", "1.17"]
+              build_tags: [ "", "appsec" ]
+      - test-outliers:
+          matrix:
+            parameters:
+              gover: ["1.14", "1.15", "1.16", "1.17"]
```

Comment:
<3 testing with different Go versions.

Q: Does this PR imply we're no longer supporting Go 1.12?